docs/source/torch.rst

```diff
@@ -337,6 +337,7 @@ Pointwise Ops
     negative
     nextafter
     polygamma
+    positive
```

Comment:
Since positive is also a method, add an entry to tensors.rst (https://github.com/pytorch/pytorch/blob/master/docs/source/tensors.rst), too